tbc/views.py

```diff
@@ -1197,20 +1197,19 @@ def ConvertNotebook(request, notebook_path=None):
     notebook_name = notebook_name[0].split(".")[0]
     path = path.split("/")[0:-1]
     path = "/".join(path)+"/"
-    os.chdir(path)
-    try:
-        changed_time = time.ctime(os.path.getctime(path+notebook_name+".html"))
-        modified_time = time.ctime(os.path.getctime(path+notebook_name+".ipynb"))
-        if(changed_time > modified_time):
-                template = path+notebook_name+".html"
-                return render_to_response(template, {})
-        else:
-                os.popen("ipython nbconvert --to html \""+path+notebook_name+".ipynb\"")
-                template = path+notebook_name+".html"
-                return render_to_response(template, {})
-    except:
-        os.popen("ipython nbconvert --to html \""+path+notebook_name+".ipynb\"")
-        template = path+notebook_name+".html"
+    template = path+notebook_name+".html"
+    notebook_html = str(path+notebook_name+".html")
+    notebook_ipynb =str(path+notebook_name+".ipynb")
```

Comment:
Is it possible to use os.path.join here as well? same goes for  lines 1200, 1201